src/main/java/com/rog/EShop/properties/ApplicationProperties.java

```diff
@@ -0,0 +1,12 @@
+package com.rog.EShop.properties;
+
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ApplicationProperties {
```

Comment:
this class currently do nothing. Remove it. When you will need it, you can create new class. This is bad practice to keep trash classes in project.